src/storage/QueryBaseProcessor.inl

```diff
@@ -263,7 +402,10 @@ QueryBaseProcessor<REQ, RESP>::asyncProcessBucket(Bucket bucket) {
         std::vector<OneVertexResp> codes;
         codes.reserve(b.vertices_.size());
         for (auto& pv : b.vertices_) {
-            codes.emplace_back(pv.first, pv.second, processVertex(pv.first, pv.second));
+            FilterContext filterContext;
```

Comment:
why not move parameter FilterContext to processVertex?

Reply:
Very good suggestion